model/task_lifecycle.go

```diff
@@ -1283,12 +1284,29 @@ func updateVersionStatus(v *Version) (string, error) {
 	return versionStatus, nil
 }
 
+// UpdatePatchStatus updates the status of a patch. For PR patches, the build associated
+// with the passed in buildVariant has its GitHub status updated.
 func UpdatePatchStatus(p *patch.Patch, versionStatus, buildVariant string) error {
 	patchStatus, err := evergreen.VersionStatusToPatchStatus(versionStatus)
 	if err != nil {
 		return errors.Wrapf(err, "getting patch status from version status '%s'", versionStatus)
 	}
 
+	if p.IsGithubPRPatch() && !evergreen.IsFinishedPatchStatus(patchStatus) {
```

Comment:
Why do we do this before the check at 1310? If that's true then I would think that this shouldn't be sent.

Reply:
I guess I just don't know why we moved it. Was this the change to fix restarts?

Reply:
Oh I see -- if we send this by build variant, then should this just be part of UpdateBuildStatus instead of UpdatePatchStatus? Doesn't seem right for it to be related to patch.